docs/interceptors.md

```diff
@@ -52,7 +52,7 @@ response.body().close();
 The URL `http://www.publicobject.com/helloworld.txt` redirects to `https://publicobject.com/helloworld.txt`, and OkHttp follows this redirect automatically. Our application interceptor is called **once** and the response returned from `chain.proceed()` has the redirected response:
 
 ```
-INFO: Sending request http://www.publicobject.com/helloworld.txt on null
+INFO: Sending request https://www.publicobject.com/helloworld.txt on null
```

Comment:
Sorry, I missed this in previous review. Given documentation below, I think this line should be reverted.